src/components/PrizePage/Prizes.js

```diff
@@ -81,35 +121,66 @@ export default function Prizes() {
 	const hideImg = useMediaQuery(theme.breakpoints.down('xs'));
 
 	return (
-		<Container maxWidth='md'>
-			<TableContainer>
-				<Table aria-label="prize table">
-					<TableHead>
-						<TableRow>
-							{ hideImg ? null : <TableCell></TableCell> }
-							<TableCell align='center' className={classes.header}>Track</TableCell>
-							<TableCell align='center' className={classes.header}>Prize</TableCell>
-						</TableRow>
-					</TableHead>
-					<TableBody>
-						{allPrizes.map(row =>
-							<TableRow key={row.track}>
-								{ hideImg ?
-									null :
-									<TableCell align='center' width='20%'>
-										<img src={row.img} height='60px'/>
+		<>
+			<Container maxWidth='md'>
+				<Grid maxWidth='md' container spacing={3} className={classes.prizeGrid}>
+					{allPrizes.map(prize =>
+						<Grid item xs={12} sm={4} key={prize.track}>
+							<Box className={classes.prizeBox}>
```

Comment:
Instead of making a class just for this, you could say
```js
<Box display='flex' justifyContent='center' flexDirection='column'>
```

Box will automatically generate a class for this combo, and the CSS properties may be more readable when specified inline.

You could probably do this for imageBox and prizeCard too, though use your best judgement w.r.t. readability. (too long of a Box tag would reduce readability)